src/Controls/tests/TestCases/Issues/Issue22914.xaml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Maui.Controls.Sample.Issues.Issue22914">
+    <VerticalStackLayout BackgroundColor="Orange"
+                         x:Name="VerticalStackLayout1">
+
+
+        <ContentView BackgroundColor="Gray"
+                     x:Name="ContentView1">
+            <Label Text="You should be able to read after tapping the button"
+                   TextColor="Gray"/>
+        </ContentView>
+
+        <Label HeightRequest="100"
+               Background="Green"
```

Comment:
Here, should use BackgroundColor. If not, after tapping the Button and setting it to null, will not impact to the Background.
![Issue22914Test png](https://github.com/dotnet/maui/assets/6755973/21c11d01-0d1a-4476-95c4-d0f6b14d6ac1)